metrics/tarantool/memtx.lua

```diff
@@ -0,0 +1,129 @@
+            memtx_tx_stat.mvcc.trackers.total)
+
+    collectors_list.memtx_tx_mvcc_conflicts_max =
+        utils.set_gauge('memtx_tx_mvcc_conflicts_max', 'Maximum bytes allocated for conflicts per transaction',
+            memtx_tx_stat.mvcc.conflicts.max)
+    collectors_list.memtx_tx_mvcc_conflicts_avg =
+        utils.set_gauge('memtx_tx_mvcc_conflicts_avg', 'Average for all transactions (total / number of transactions)',
+            memtx_tx_stat.mvcc.conflicts.avg)
+    collectors_list.memtx_tx_mvcc_conflicts_total =
+        utils.set_gauge('memtx_tx_mvcc_conflicts_total', 'Bytes allocated for conflicts in total',
+            memtx_tx_stat.mvcc.conflicts.total)
+
+
+    collectors_list.memtx_tx_mvcc_tuples_tracking_stories_count =
+        utils.set_gauge('memtx_tx_mvcc_tuples_tracking_stories_count',
+            'Number of `tracking` tuples / number of tracking stories.',
+            memtx_tx_stat.mvcc.tuples.tracking.stories.count)
+    collectors_list.memtx_tx_mvcc_tuples_tracking_stories_total =
+        utils.set_gauge('memtx_tx_mvcc_tuples_tracking_stories_total',
+            'Amount of bytes used by stories `tracking` tuples',
+            memtx_tx_stat.mvcc.tuples.tracking.stories.total)
+
+    collectors_list.memtx_tx_mvcc_tuples_tracking_retained_count =
+        utils.set_gauge('memtx_tx_mvcc_tuples_tracking_retained_count',
+            'Number of retained `tracking` tuples / number of stories',
+            memtx_tx_stat.mvcc.tuples.tracking.retained.count)
+    collectors_list.memtx_tx_mvcc_tuples_tracking_retained_total =
+        utils.set_gauge('memtx_tx_mvcc_tuples_tracking_retained_total',
+            'Amount of bytes used by retained `tracking` tuples',
+            memtx_tx_stat.mvcc.tuples.tracking.retained.total)
+
+
+    collectors_list.memtx_tx_mvcc_tuples_used_stories_count =
+        utils.set_gauge('memtx_tx_mvcc_tuples_used_stories_count', 'Number of `used` tuples / number of stories',
+            memtx_tx_stat.mvcc.tuples.used.stories.count)
+    collectors_list.memtx_tx_mvcc_tuples_used_stories_total =
+        utils.set_gauge('memtx_tx_mvcc_tuples_used_stories_total', 'Amount of bytes used by stories ``used`` tuples',
+            memtx_tx_stat.mvcc.tuples.used.stories.total)
+
+    collectors_list.memtx_tx_mvcc_tuples_used_retained_count =
+        utils.set_gauge('memtx_tx_mvcc_tuples_used_retained_count',
+            'Number of retained `used` tuples / number of stories',
+            memtx_tx_stat.mvcc.tuples.used.retained.count)
+    collectors_list.memtx_tx_mvcc_tuples_used_retained_total =
+        utils.set_gauge('memtx_tx_mvcc_tuples_used_retained_total', 'Amount of bytes used by retained `used` tuples',
+            memtx_tx_stat.mvcc.tuples.used.retained.total)
+
+
+    collectors_list.memtx_tx_mvcc_tuples_read_view_stories_count =
+        utils.set_gauge('memtx_tx_mvcc_tuples_read_view_stories_count',
```

Comment:
If it's 'count', the metric should be a counter, not gauge 
Also, please run promtool (look at the step in CI) to make sure you named metrics correctly

Reply:
I added a label and got rid of the suffix `count` completely. So metric is keeping be gauge by its nature.